### DISCUSSION TITLE
There is a problem with PasswordInput

### DISCUSSION THREAD

**Discussion by afgprogrammer** [mantinedev/mantine#2260] [Q&A] [answered]
I want to change the height of the inputs like this:
```
<MantineProvider
      theme={{
        ...
        components: {
          InputWrapper: {
            height: 50,
            styles: {
              root: {
                input: {
                  height: 42,
                  lineHeight: "42px",
                },
              },
              label: {
                marginBottom: 5,
              },
            },
          },
        },
      }}
    >
      <App />
    </MantineProvider>
```

Everything is fine but the PasswordInput wrapper height is not changing.

can someone help me with this?

Screenshot:
<img width="492" alt="Screen Shot 2022-08-28 at 2 35 05 PM" src="https://user-images.githubusercontent.com/17293422/187068579-bb0bce4e-06df-4ebf-abf4-ae3194d3d5e0.png">

**Comment by rtivital** [comment, marked as answer]
https://codesandbox.io/s/elastic-leaf-pozs85?file=/src/App.tsx

**Reply by afgprogrammer** [reply]
Thank you 🙏
